src/transformers/__init__.py

```diff
@@ -44,6 +44,7 @@
 from .file_utils import (
     _LazyModule,
     is_flax_available,
+    is_quantization_available,
```

Comment:
I'm not a fan of the `is_quantization_available` name as it sounds as if it's all quantization capabilities - when it's actually a flag for the `pytorch_quantization` package. Would `is_pytorch_quantization_available` work for you?

Reply:
Sounds good.